corehq/apps/integration/kyc/tests/test_views.py

```diff
@@ -46,6 +55,170 @@ def _make_request(self, is_logged_in=True):
             self.client.login(username=self.username, password=self.password)
         return self.client.get(self.endpoint)
 
+
+class TestKycConfigurationView(BaseTestKycView):
+    urlname = KycConfigurationView.urlname
+
+    def test_not_logged_in(self):
+        response = self._make_request(is_logged_in=False)
+        self.assertEqual(response.status_code, 404)
```

Comment:
Nit: Now that we are using pytest, we can use Pythonic asserts.
```suggestion
        assert response.status_code == 404
```

Reply:
I've gone ahead and replaced most of the asserts in this test file with Pythonic asserts in [fd522a5](https://github.com/dimagi/commcare-hq/pull/35765/commits/fd522a52d24f844987ac5ff1db84d8f7ae75db79) (all apart from the `self.assertRedirects` one).